class/defaults.yml

```
@@ -101,7 +101,7 @@ parameters:
             params:
               cloud_provider: xelon
               vshn_service_level: zero
-        instance_id_pattern: '%(cluster_id)s'
+        instance_id_pattern- 'ocp-cluster-%(cluster_id)s-%(vshn_service_level)s'
```

```suggestion
        instance_id_pattern: 'ocp-cluster-%(cluster_id)s-%(vshn_service_level)s'
```

Shoudn't  it be that way?

---

Yes that is why all tests fail.